INSS.FIP.Web.UnitTests/ValidationTests/SearchParametersViewModelValidationTests.cs

```diff
@@ -2,20 +2,30 @@
 using INSS.FIP.Services;
 using INSS.FIP.Web.ViewModels;
 using Xunit;
-
+using INSS.FIP.Web.ViewModels;
```

Comment:
remove unnecessary namespace  (INSS.FIP.Web.ViewModels added twice)

Reply:
Removed in my last commit .